compose/snippets/src/androidTest/java/com/example/compose/snippets/deviceconfigurationoverride/DeviceConfigurationOverrideSnippets.kt

```diff
@@ -1,65 +0,0 @@
-/*
- * Copyright 2024 The Android Open Source Project
```

Comment:
@alexvanyo is this correct that its deleting this file?

Reply:
Hmm that looks wrong - I merged those snippets in #232 directly into `main`, and it looks like it the file isn't in `latest` right now.

Reply:
fixed